src/modules/tls/tls_mod.c

```diff
@@ -496,3 +527,59 @@ int mod_register(char *path, int *dlflags, void *p1, void *p2)
 
 	return 0;
 }
+
+
+
+#ifndef OPENSSL_NO_ENGINE
+/*
+ * initialize OpenSSL engine in child process
+ * PKCS#11 libraries are not guaranteed to be fork() safe
+ *
+ */
+static int engine_init()
+{
+	/*
```

Comment:
As discussed, this should be either in the module README or as dedicated cfg example in the module dir.